aten/src/ATen/native/cuda/Blas.cpp

```diff
@@ -126,14 +126,14 @@ enum class Activation {
 cuda::blas::GEMMAndBiasActivationEpilogue activation_to_gemm_and_blas_arg(Activation a) {
   switch (a) {
     case Activation::None:
-      return cuda::blas::GEMMAndBiasActivationEpilogue::None;
+      return cuda::blas::GEMMAndBiasActivationEpilogue::BIAS;
```

Comment:
why is this change?

Reply:
Well for "Activation::None" we still want do Bias addition and "Activation RELU" means "do  Bias and Relu". But for _int_mm we want to do nothing. Hence `GEMMAndBiasActivationEpilogue` gained a `NONE` option. I can update the `Activation` enum as well, but technically `bias` isn't an activation.